web/admin/src/Admin.js

```diff
@@ -254,6 +361,10 @@ export default class Admin extends PureComponent {
     tokenRef.remove()
   }
 
+  updateExportIsDisabled = bool => {
```

Comment:
I don't think we should have a separate state variable for this.  We can determine whether the export buttons are disabled from the state of the current session.

Reply:
That was my initial thought but we don't have a listener set up on the homepage for the public session data. Was 50-50 on which option was better

Reply:
A better approach is to "lift state up" https://reactjs.org/docs/lifting-state-up.html rather than having duplicates of state (since we can know the value of `exportIsDisabled` by knowing the state of the trivia session) and making sure we update that copy in all the right places.
Since 👆 uses a prop function to effectively send state up the component tree, it would instead be best to have a parent component hold the state of a trivia session and listen to firebase, and pass down the state as props.

Or just put the export buttons in PresentationDriver and leave all the state where it is.

This can be refactored separately from this PR, however.